vyper/codegen/stmt.py

```diff
@@ -138,7 +138,6 @@ def parse_Call(self):
             "append",
             "pop",
         ):
-            # TODO: consider moving this to builtins
```

Comment:
I think this is an outdated comment since `pop` and `dynarray` were moved from builtins to codegen.